tests/suites/test_suite_ecp.function

```diff
@@ -1295,3 +1295,93 @@ exit:
     mbedtls_mpi_free(&expected_n);
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void ecp_quasi_reduction(char *input_N,
+                         char *input_A,
+                         char *result)
+{
+    mbedtls_mpi_uint *A = NULL;
+    mbedtls_mpi_uint *N = NULL;
+    mbedtls_mpi_uint *res = NULL;
+    size_t limbs_A;
+    size_t limbs_N;
+    size_t limbs_res;
+
+    mbedtls_mpi_mod_modulus m;
+    mbedtls_mpi_mod_modulus_init(&m);
+
+    TEST_EQUAL(mbedtls_test_read_mpi_core(&A,   &limbs_A,   input_A), 0);
+    TEST_EQUAL(mbedtls_test_read_mpi_core(&N,   &limbs_N,   input_N), 0);
+    TEST_EQUAL(mbedtls_test_read_mpi_core(&res, &limbs_res, result),  0);
+
+    size_t limbs = limbs_N;
+    size_t bytes = limbs * sizeof(mbedtls_mpi_uint);
+
+    TEST_EQUAL(limbs_A,   limbs);
+    TEST_EQUAL(limbs_res, limbs);
+
+    TEST_EQUAL(mbedtls_mpi_mod_modulus_setup(
+                   &m, N, limbs,
+                   MBEDTLS_MPI_MOD_REP_MONTGOMERY), 0);
+
+    TEST_EQUAL(mbedtls_ecp_quasi_reduction(A, &m), 0);
+    ASSERT_COMPARE(A, bytes, res, bytes);
+
```

Comment:
Would it add any value to test for aliased input?  

~~~~~~
TEST_EQUAL(mbedtls_ecp_quasi_reduction(&m->p, &m), 0);
~~~~~~

Reply:
That use is not really meaningful and `&m->p` supposed to be `const` anyway, I don't think we should test this.